spec/apis/path-to-active-when.js

```diff
@@ -0,0 +1,145 @@
+import { pathToActiveWhen } from "single-spa";
+import { toDynamicPathValidatorRegex } from "../../src/applications/apps";
+
+describe(`pathToActiveWhen`, () => {
+  it(`allows you to create your own activeWhen functions using the single-spa public api`, () => {
```

Comment:
This is a new test that verifies that pathToActiveWhen is available as a public api. The tests below are ones that already existed that verify the function's behavior in more depth.